web/wp-content/themes/borealis-2021/src/sass/style.scss

```
@@ -351,6 +351,12 @@ legend {
   overflow: hidden;
 }
 
+.backdrop {
+  -webkit-backdrop-filter: blur(5px);
+  backdrop-filter: blur(5px);
+  background-color: rgba(255, 255, 255, 0.1);
```

Fun fact!  you can also write this as a hex!
```suggestion
  background-color: #FFFFFF1A;
```
Just dropping some knowledge not requesting a change.  I actually think this is more readable and better